src/App.css

```diff
@@ -74,6 +73,15 @@ input {
 
 @media (max-width: 720px) {
   .Results {
-    width: 240px;
```

Comment:
Please re-check the CSS and see if it's clashing with the current 'Results' or 'Results-card' div-s because the widths were added to have responsive design.